blog/authors.yml

```diff
@@ -21,3 +21,9 @@ lucasfernog:
   title: Tauri Cofounder
   url: https://github.com/lucasfernog
   image_url: ./author_images/lucasfernog.jpg
+
+wusyong:
+  name: Wu Yu Wei
+  title: Tauwi dewewoper UwU
```

Comment:
```suggestion
  title: Tauri Core
```

;)

Reply:
Sorry about all these typo. I just wanna saw some previous first. I'll fix them in a moment.